code/modules/mob/mob_grab.dm

```diff
@@ -169,6 +169,10 @@
 					if(do_mob(assailant, affecting, 150))
 						A.visible_message(SPAN_WARNING("[A] falls unconscious..."), FONT_LARGE(SPAN_DANGER("The world goes dark as you fall unconscious...")))
 						A.Paralyse(20)
+		else if(istype(affecting, /mob/living/simple_animal))
+			var/mob/living/simple_animal/SA = affecting
+			if(SA.stat != DEAD)
+				SA.health -= 1
```

Comment:
Maybe make them lose a % of maxhealth instead?

Reply:
nahhhh, i'd prefer to keep it as one HP, because creatures with large amounts of HP like Krampus and the Queen spider shouldn't even get dented by this damage, imo